cmd/sidecar/worker.go

```diff
@@ -739,15 +740,36 @@ func (worker *inboundWorker) validateFilesHeaders(req *inboundRequest, ifiles *[
 	return true
 }
 
+func (worker *inboundWorker) validateFnCtxHeaders(req *inboundRequest, fnCtxStr string) (enginerefactor.FunctionContext, bool) {
+	hdr := "Direktiv-Function-Context"
+	data, err := base64.StdEncoding.DecodeString(fnCtxStr)
+	if err != nil {
+		worker.reportValidationError(req, http.StatusBadRequest, fmt.Errorf("invalid %s [%d]: %w", hdr, 0, err))
```

Comment:
What is the meaning of the `[%d]` `0` in this message? Can we remove it? Same question for the similar error message that appears in this function.